docs/docs/reference/migration.md

```diff
@@ -3,51 +3,80 @@
 This document describes breaking changes and migrations between Constellation releases.
 Use [`constellation config migrate`](./cli.md#constellation-config-migrate) to automatically update an old config file to a new format.
 
+## Migrations to v2.19.1
+
+### Azure
+
+* During the upgrade, security rules are migrated and the old ones are recommended to be cleaned up manually by the user. This step is recommended but not strictly required. The below script shows how to programatically delete the old rules through the Azure CLI:
```

Comment:
I'd recommend to not make this optional, because we could not change the security rule name back and remove the priority offset in Terraform otherwise.